stripe/src/main/java/com/stripe/android/googlepay/StripeGooglePayContract.kt

```diff
@@ -128,3 +129,31 @@ internal class StripeGooglePayContract :
         }
     }
 }
+
+internal fun Status.getErrorMessage(): String? {
```

Comment:
Totally fine, but might be more idiomatic as a property `internal val Status.errorMessage get() = `